### DISCUSSION TITLE
Auto-filling content to a tag column through copy/paste or excel-like dragging

### DISCUSSION THREAD

**Discussion by aparente** [RafaelGB/obsidian-db-folder#873] [Ideas]
I have a database with a tag column, sometimes I want to add 5+ tags to a column and would love to be able to do this without manually selecting each tag. 

I could see a couple ways of doing this:

- Simple (in theory): Just allowing for copy/paste in tag columns
- Most valuable: Enabling excel-like column dragging for replicating or iterating fields in any column.

**Comment by RafaelGB** [comment]
Hello @aparente , could be both nice solutions but it requires some work.

I leave it tagged to be evaluated in the future, right now I am in full refactoring so that the project is much easier to scale and in the future it could be viable
